manager/src/main/java/com/redhat/service/smartevents/manager/models/Bridge.java

```diff
@@ -71,6 +72,12 @@ public class Bridge extends ManagedDefinedResource<BridgeDefinition> {
     @Column(name = "region", nullable = false, updatable = false)
     private String region;
 
+    @Column(name = "bridge_error_id")
```

Comment:
@manstis I personally would just store the `status_message` directly on the object here.

Reply:
I am tempted not to; as it does not lend itself to i18n.